source/PlayerInfo.cpp

```diff
@@ -7,10 +7,7 @@ Foundation, either version 3 of the License, or (at your option) any later versi
 
 Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
 WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
-PARTICULAR PURPOSE. See the GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License along with
-this program. If not, see <https://www.gnu.org/licenses/>.
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
```

Comment:
Looks like some coding style fixes (from https://github.com/endless-sky/endless-sky/pull/7216 and another PR) got lost in this PR (probably due to a failed rebase, or by overwriting the file with an older version).
Please make sure that the work from those other PRs doesn't get lost.

Reply:
Okay, have manually fixed those lines to match with Master. Is that okay? The diff-with-master should now only be my changes.

Reply:
Yes, that is fine. Thanks!